cql/ManageCommonAbnormality.cql

```diff
@@ -462,7 +462,8 @@ define ErrataRecommendation:
         short: 'Colposcopy',
         date: Today(),
         details: {
-          'Colposcopy is recommended.'
+          'Colposcopy is recommended.',
+          'If a patient has a negative HPV test or co-test following a low-grade result for which colposcopy was previously recommended but not performed, repeating an HPV test or cotest in 1 year is acceptable. (Perkins 2019 Erratum)'
```

Comment:
```suggestion
          'If a patient has a negative HPV test or cotest following a low-grade result for which colposcopy was previously recommended but not performed, repeating an HPV test or cotest in 1 year is acceptable.'
```
Comments about recommendation text:
1. `co-test` --> `cotest` to align with other recommendations
2. Is there a reason the source document is referenced in this recommendations, when it isn't referenced in most of the other recommendation statements? Should this be added to the "Group" field instead?

Reply:
I believe the Perkins 2019 Erratum is different than the 2023 erratum, correct? In that case, it seems confusing to list "Perkins 2019 Erratum" in the recommendation itself but then also list "2023 JLGTD Addendum" in the source section of the recommendation.

Reply:
1. This will be addressed, as the recommendation text will be updated to the following:

> If the previous cytology result was not high grade, and the patient undergoes repeat testing with HPV testing or cotesting instead of colposcopy: colposcopy is recommended if the result on repeat testing indicates a second consecutive HPV-positive result and/or persistent cytologic abnormality (ASC-US or a more severe cytologic interpretation); repeat HPV testing or cotesting in 1 year is acceptable if the result on repeat testing is HPV negative or cotest negative.

2. The group field should not need to be updated any more, since the text excerpt in our recommendation will be coming directly from the erratum, which is already listed in the group field.